index.js

```diff
@@ -405,8 +429,10 @@ PersonaClient.prototype.obtainToken = function (id, secret, callback) {
                         }
                     };
                 var req = _this.http.request(options, function (resp) {
+                    _this.debug(JSON.stringify(resp));
+                    var str = '';
```

Comment:
There's a complete all over the map mix of single and double quotes all through the code.  We probably need to decide what we want to do about that (not today).